commit d5ce90ccc2eabde0166aa718e89becb17ff6f763 (refs/changes/33/4033/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-05-30T22:14:36-07:00 (1 year, 4 months ago)
    
    TRITON-380 work around logadm.conf "/var/log/*.log" entry in newer origins to not lose logs
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index bda632a..6bfa4a2 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit bda632a69ff44114f9faf689a5cd4af1fe578d66
+Subproject commit 6bfa4a298d753685c6e083d10b70b5abdc39f66f
